eslint.config.mjs

```diff
@@ -2,9 +2,11 @@ import eslintPluginJest from "eslint-plugin-jest";
 import eslintPluginJs from "@eslint/js";
 import eslintPluginStylistic from "@stylistic/eslint-plugin";
 import globals from "globals";
+import importPlugin from "eslint-plugin-import";
```

Comment:
Can we keep a consitant variable name pattern for plugins?

I suggest `eslintPluginImport`.

Reply:
sure :-)